installer/bundle/bundle_skel.sh

```diff
@@ -784,15 +767,19 @@ case "$installMode" in
         if [ $scx_installed -ne 0 -a $omi_installed -ne 0 ]; then
             echo "Installing OMS agent ..."
 
-            pkg_add_list $OMI_PKG omi
-            pkg_add_list $SCX_PKG scx
             pkg_add_list $OMS_PKG omsagent
 
             python_ctypes_installed
             if [ $? -eq 0 ]; then
                 pkg_add_list $DSC_PKG omsconfig
             fi
 
+            # Install SCX (and OMI)
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            SCX_STATUS=$?
+
             # Now actually install of the "queued" packages
             if [ -n "${add_list}" ]; then
                 if [ "$INSTALLER" = "DPKG" ]; then
```

Comment:
So historically, we thought that bundling packages was more reliable for installation. But then we came up with cases where we were better off without bundled installations and having a recovery procedure for SCX.

Which is better for OMS? Is there a reason we should keep bundled package installation, or should we go back to individual package installation, like it used to be? (We changed it to be consistent with SCX because at the time we thought that was best ...)

I don't know the answer, but wanted you to think about it, so I'm bringing it up ... I'm likely to accept any well thought out, reasoned answer. We do have a little evidence, though, that bundled installations don't always pan out as we hoped they would (thus the recovery procedure).